doc/dev/design/FABRID.rst

```diff
@@ -0,0 +1,451 @@
+The AS network operator configures which global FABRID policies are supported for the local AS and can add additional local FABRID
+policies that are valid for this AS.
+The FABRID policy identifiers are provided to the endhosts inside of the PCB or as an detachable PCB extension.
+The FABRID policy descriptions on the other hand have to be requested from the local control service which will then either return
+the cached FABRID policy description or query the policy description from the remote AS.
+That way we can minimize the overhead that the FABRID polices cause to the PCB size.
+A source endhost can then select a path together with FABRID policies and forward the FABRID packet over this path to a destination endhost.
+The on-path border routers will then add some proof of transit such that the destination endhost can later verify that the packet indeed followed the intended path.
+
+Our proposed design and implementation of FABRID allows for incremental deployment at router- and AS-level, i.e., some AS operators may want to
+deploy FABRID while others do not, and those who do may only want to deploy it on a subset of border routers.
+This allows for a smooth migration where an AS can test-wise update some border routers and test that nothing breaks.
+However, this could lead to the situation where an end host does not have paths available for which all on-path ASes support FABRID.
+In such a situation the end host can still send its traffic along that path, but without any of the guarantees provided by FABRID for those ASes.
+
+Since each AS can create their own local FABRID policies, end hosts have to learn them.
+In our design, end hosts have to learn them from their local AS, and the local AS has to learn them remote AS, similar to how SCION path retrieval is implemented.
+Those policies are only fetched on demand by the local control service and will be cached till end of their validity.
+This allows for better scalability for the FABRID policies because an AS does not have to learn all FABRID policies from all other ASes.
+Even though the beacon had to be adapted, the size increase is negligible.
+
+The design document specifies:
+
+- The header design
+    Specification of the two new Hop-by-Hop extension options, their fields and how they are computed.
+- The data plane
+    Describes how the FABRID packets are processed and forwarded at the border routers, how the endhosts can send FABRID packets,
+    and the path validation for the destination endhost.
+- The control plane
+    Describes how the FABRID policies are defined, additional beaconing information, and new FABRID control service endpoints which are used for
+    communicating the FABRID policies to the endhosts and remote ASes.
+- Configuration
+    Describes how the border router and the control service have to be configured such that they are able to process FABRID packets.
```

Comment:
Please also add the other sections.
Rationale, implementation, ...

Reply:
At the moment the list describes all sub sections of the proposal section. (and is inside of the proposal section). If we should describe more then that it might make sense to move this out of the proposal section and create some sort of "table of contents". If I compare your suggestion with the other documents I don't see that they did something similar like this.

Reply:
Both options work for me, adding those points here or in a dedicated table of contents section.
We don't need to strictly follow the other documents :)